shared/js/ui/views/hamburger-menu.es6.js

```diff
@@ -53,9 +54,8 @@ HamburgerMenu.prototype = window.$.extend({},
 
         _handleBrokenSiteClick: function (e) {
             e.preventDefault()
-
-            let url = encodeURIComponent(this.model.tabUrl)
-            browserUIWrapper.openExtensionPage(`/html/feedback.html?broken=1&url=${url}`)
+            this.$el.addClass('is-hidden')
+            this.pageView.views.site.showBreakageForm('reportBrokenSite')
```

Comment:
Passing a param to showBreakageForm so that the page doesn't automatically reload when the form is submitted, unless the user came to the form from toggling privacy protection (in which case we want to keep behavior consistent with what it was previously -- toggle privacy protection and site reloads with new setting).